code/modules/mob/dead/observer/observer.dm

```diff
@@ -446,6 +447,29 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			else
 				A << "This mob is not located in the game world."
 
+/mob/dead/observer/verb/change_view_range()
+	set category = "Ghost"
+	set name = "Modify View Range"
```

Comment:
I might suggest naming this `zoom` or `set zoom`. There is a downside to longer names of verbs in that it fits less on the panel because it makes them all wider.